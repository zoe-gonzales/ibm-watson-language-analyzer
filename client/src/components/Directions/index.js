import React from 'react';

const Directions = ({ dirs }) => {
    return dirs.map(dir => <h5 key={dir.id}>{dir.text}</h5>);
}

export default Directions;