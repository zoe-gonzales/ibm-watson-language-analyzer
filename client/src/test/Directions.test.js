import React from 'react';
import Directions from '../components/Directions';
import renderer from 'react-test-renderer';

it('renders Directions component correctly', () => {
    const steps = [
        'Step 1',
        'Step 2',
        'Step 3'
    ];
    const component = renderer
    .create(<Directions steps={steps} />)
    .toJSON();
    expect(component).toMatchSnapshot();
});