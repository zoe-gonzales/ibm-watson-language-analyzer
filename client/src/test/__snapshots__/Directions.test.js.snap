// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Directions component correctly 1`] = `
Array [
  <h5>
    Step 1
  </h5>,
  <h5>
    Step 2
  </h5>,
  <h5>
    Step 3
  </h5>,
]
`;
